src/sounds.cpp

```diff
@@ -1594,6 +1602,7 @@ void sfx::do_footstep() { }
 void sfx::do_danger_music() { }
 void sfx::do_vehicle_engine_sfx() { }
 void sfx::do_vehicle_exterior_engine_sfx() { }
+std::pair<std::string, std::string> sfx::get_engine_sound( std::string, vehicle *, bool ) { return nullptr; }
```

Comment:
This doesn't return a pointer (and shouldn't)
```suggestion
std::pair<std::string, std::string> sfx::get_engine_sound( std::string, vehicle *, bool ) { return { std::string(), std::string() }; }
```
or
```suggestion
std::pair<std::string, std::string> sfx::get_engine_sound( std::string, vehicle *, bool ) { return { "", "" }; }
```
or preferrably this, but I'm not sure if it will work
```suggestion
std::pair<std::string, std::string> sfx::get_engine_sound( std::string, vehicle *, bool ) { return {}; }
```